projects/kit/components/input-phone-international/const/countries.ts

```diff
@@ -1,7 +1,9 @@
-import {TuiCountryIsoCode} from '../country-iso-code';
+import {TuiCountryIsoCode} from '@taiga-ui/i18n';
 import {Country} from '../interfaces/country';
 
-// TODO: i18n
+/**
+ * TODO: remove property `name` in 3.0
```

Comment:
Looks like in 3.0 it would become `COUNTRIES_MASKS` with just a `Record<TuiCountryIsoCode, string>`

Reply:
i introduced const `COUNTRIES_MASKS ` and mark `COUNTRIES` as deprecated